editor/filesystem_dock.cpp

```diff
@@ -3965,13 +3926,13 @@ MenuButton *FileSystemDock::_create_file_menu_button() {
 
 	PopupMenu *p = button->get_popup();
 	p->connect(SceneStringName(id_pressed), callable_mp(this, &FileSystemDock::_file_sort_popup));
-	p->add_radio_check_item(TTR("Sort by Name (Ascending)"), FILE_SORT_NAME);
-	p->add_radio_check_item(TTR("Sort by Name (Descending)"), FILE_SORT_NAME_REVERSE);
-	p->add_radio_check_item(TTR("Sort by Type (Ascending)"), FILE_SORT_TYPE);
-	p->add_radio_check_item(TTR("Sort by Type (Descending)"), FILE_SORT_TYPE_REVERSE);
-	p->add_radio_check_item(TTR("Sort by Last Modified"), FILE_SORT_MODIFIED_TIME);
-	p->add_radio_check_item(TTR("Sort by First Modified"), FILE_SORT_MODIFIED_TIME_REVERSE);
-	p->set_item_checked(file_sort, true);
+	p->add_radio_check_item(TTR("Sort by Name (Ascending)"), static_cast<int>(FileSortOption::FILE_SORT_NAME));
+	p->add_radio_check_item(TTR("Sort by Name (Descending)"), static_cast<int>(FileSortOption::FILE_SORT_NAME_REVERSE));
+	p->add_radio_check_item(TTR("Sort by Type (Ascending)"), static_cast<int>(FileSortOption::FILE_SORT_TYPE));
+	p->add_radio_check_item(TTR("Sort by Type (Descending)"), static_cast<int>(FileSortOption::FILE_SORT_TYPE_REVERSE));
+	p->add_radio_check_item(TTR("Sort by Last Modified"), static_cast<int>(FileSortOption::FILE_SORT_MODIFIED_TIME));
+	p->add_radio_check_item(TTR("Sort by First Modified"), static_cast<int>(FileSortOption::FILE_SORT_MODIFIED_TIME_REVERSE));
```

Comment:
"First Modified" doesn't really speak to me for what this is describing. What terminology is used in similar software for this? I know GitHub uses "least recently updated", which is more verbose but much clearer IMO.

Reply:
I've no problem to change it. That text is already being used [here](https://github.com/godotengine/godot/blob/a0d1ba4a3d1760f48ef3297a6299ee3dbc1260e1/editor/filesystem_dock.cpp#L3973)
As I mentioned [here](https://github.com/godotengine/godot/pull/97344#discussion_r1775925352), changing it will require translation changes and sadly I don't know if there is a process for that.

Reply:
Right, if this is pre-existing terminology we can keep it, no reason to block this PR to start another bikeshedding :D

Regarding translations, the process is that at some point I'll switch Weblate to track the `master` branch so the change would be propagated there. For now it's tracking the `4.3` branch, so we can update 4.3.x translations. As long as we don't need to backport the PR to 4.3, there's no concern about updating translation strings.